tests/definitions.py

```diff
@@ -17,8 +17,19 @@
 import pytest
 import datetime
 
+import gt4py.backend as gt_backend
 import gt4py.utils as gt_utils
 
+ALL_BACKENDS = list(gt_backend.REGISTRY.keys())
+CPU_BACKENDS = [
+    name for name in ALL_BACKENDS if gt_backend.from_name(name).storage_info["device"] == "cpu"
```

Comment:
I know it's what we have always been doing, but maybe we could make device an attribute of the backend.